aten/src/ATen/native/native_functions.yaml

```diff
@@ -1286,6 +1286,7 @@
   dispatch:
     SparseCPU, SparseCUDA: cat_sparse
     QuantizedCPU: cat_quantized_cpu
+    NestedTensorCPU, NestedTensorCUDA: cat_nested
```

Comment:
If we land this, I guess it'd be better to bump cat support to a separate PR to keep them modular